src/index.js

```diff
@@ -46,7 +46,7 @@ Chart.Ticks = Ticks;
 Object.assign(Chart, controllers, scales, elements, plugins, platforms);
 Chart.Chart = Chart;
 
-if (typeof window !== 'undefined') {
+if (window !== undefined) {
```

Comment:
This is not equivalent. This will throw an error `Uncaught ReferenceError: window is not defined`, while the previous version will not. You can try it yourself in the console with some undefined variable `asd !== undefined`

Reply:
Thanks for reviewing @danmana